grayskull/pypi/pypi.py

```diff
@@ -552,6 +552,12 @@ def _get_metadata(self) -> dict:
         version = ""
         if self["package"]["version"].values:
             version = self.get_var_content(self["package"]["version"].values[0])
+        if name.startswith("http://"):
```

Comment:
We need to consider https protocols too. A nice but little known feature of `.startswith` is that it supports tuples of strings as an argument to try different things. That way we can write this as:

```suggestion
        if name.startswith(("http://", "https://")):
```